x-pack/plugins/snapshot_restore/public/application/components/summaries/policies/policy_feature_states_summary.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import {
+  EuiFlexItem,
+  EuiDescriptionList,
+  EuiDescriptionListTitle,
+  EuiDescriptionListDescription,
+} from '@elastic/eui';
+import { SnapshotConfig } from '../../../../../common/types';
+import { FEATURE_STATES_NONE_OPTION } from '../../../../../common/constants';
+import { CollapsibleFeatureStatesList } from '../../collapsible_lists';
+
+export const PolicyFeatureStatesSummary: React.FunctionComponent<SnapshotConfig> = ({
+  includeGlobalState,
+  featureStates,
+}) => {
+  const hasGlobalStateButNoFeatureStates = includeGlobalState && featureStates === undefined;
+  const hasNoFeatureStates = !featureStates || featureStates?.includes(FEATURE_STATES_NONE_OPTION);
+  const hasAllFeatureStates = hasGlobalStateButNoFeatureStates || featureStates?.length === 0;
+
+  return (
+    <EuiFlexItem data-test-subj="policyFeatureStatesSummary">
+      <EuiDescriptionList textStyle="reverse">
+        <EuiDescriptionListTitle>
+          <FormattedMessage
+            id="xpack.snapshotRestore.summary.policyFeatureStatesLabel"
+            defaultMessage="Include feature state {hasSpecificFeatures, plural, one {from} other {}}"
+            values={{ hasSpecificFeatures: !hasNoFeatureStates && !hasAllFeatureStates ? 1 : 0 }}
+          />
+        </EuiDescriptionListTitle>
+        <EuiDescriptionListDescription data-test-subj="value">
+          {!hasGlobalStateButNoFeatureStates && hasNoFeatureStates && (
+            <FormattedMessage
+              id="xpack.snapshotRestore.summary.policyNoFeatureStatesLabel"
+              defaultMessage="No"
+            />
+          )}
+          {hasAllFeatureStates && (
+            <FormattedMessage
+              id="xpack.snapshotRestore.summary.policyAllFeatureStatesLabel"
+              defaultMessage="From all features"
+            />
+          )}
+          {!hasNoFeatureStates && !hasAllFeatureStates && (
+            <CollapsibleFeatureStatesList featureStates={featureStates} />
```

Comment:
Sort of an aside--it doesn't seem like the bullets for the features are necessary, they would stand out prominently without them.

Are the bold settings actually larger than the label text? They seem kind of overwhelming visually.

Reply:
I've also had the feeling that they can be a bit much, but I tried to follow the same pattern that the `Data streams and indices` summary uses so that it would be visually consistent. If we were to change these representation we would need to do in both places (which I think we could do with a separate issue). What do you think @cristina-eleonora ?

Reply:
We will address this in a separate PR, I've created an issue for this https://github.com/elastic/kibana/issues/132211